test/lib/ufe/testUINodeGraphNode.py

```diff
@@ -56,34 +56,58 @@ def setUp(self):
         # Clear selection to start off
         cmds.select(clear=True)
 
-    def testUIInfo(self):
-        ball3Path = ufe.PathString.path('|transform1|proxyShape1,/Ball_set/Props/Ball_3')
-        ball3SceneItem = ufe.Hierarchy.createItem(ball3Path)
-
-        uiNodeGraphNode = ufe.UINodeGraphNode.uiNodeGraphNode(ball3SceneItem)
-
-        self.assertFalse(uiNodeGraphNode.hasPosition())
-        pos0 = uiNodeGraphNode.getPosition()
+    # Helper to avoid copy-pasting the entire test
+    def doPosAndSizeTests(self, hasFunc, setFunc, getFunc, cmdFunc):
+        # Test hasFunc and getFunc
+        self.assertFalse(hasFunc())
+        pos0 = getFunc()
         self.assertEqual(pos0.x(), 0)
         self.assertEqual(pos0.y(), 0)
+
+        # Test setFunc with vector
         pos1 = ufe.Vector2f(10, 20)
-        uiNodeGraphNode.setPosition(pos1)
-        self.assertTrue(uiNodeGraphNode.hasPosition())
-        pos2 = uiNodeGraphNode.getPosition()
+        setFunc(pos1)
+        self.assertTrue(hasFunc())
+        pos2 = getFunc()
         self.assertEqual(pos1.x(), pos2.x())
         self.assertEqual(pos1.y(), pos2.y())
-        uiNodeGraphNode.setPosition(13, 41)
-        self.assertTrue(uiNodeGraphNode.hasPosition())
-        pos3 = uiNodeGraphNode.getPosition()
+
+        # Test setFunc with scalars
+        setFunc(13, 41)
+        self.assertTrue(hasFunc())
+        pos3 = getFunc()
         self.assertEqual(pos3.x(), 13)
         self.assertEqual(pos3.y(), 41)
+
+        # Test cmdFunc
         pos4 = ufe.Vector2f(21, 20)
-        cmd = uiNodeGraphNode.setPositionCmd(pos4)
+        cmd = cmdFunc(pos4)
         cmd.execute()
-        self.assertTrue(uiNodeGraphNode.hasPosition())
-        pos5 = uiNodeGraphNode.getPosition()
+        self.assertTrue(hasFunc())
+        pos5 = getFunc()
         self.assertEqual(pos4.x(), pos5.x())
         self.assertEqual(pos4.y(), pos5.y())
 
+    def testPosition(self):
+        ball3Path = ufe.PathString.path('|transform1|proxyShape1,/Ball_set/Props/Ball_3')
+        ball3SceneItem = ufe.Hierarchy.createItem(ball3Path)
+
+        uiNodeGraphNode = ufe.UINodeGraphNode.uiNodeGraphNode(ball3SceneItem)
+        self.doPosAndSizeTests(uiNodeGraphNode.hasPosition, uiNodeGraphNode.setPosition,
+            uiNodeGraphNode.getPosition, uiNodeGraphNode.setPositionCmd)
+
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '4100', 'Test for preview version 4.0.100 and later')
```

Comment:
This test is run with Ufe v0.5 as well - right? 

Should the string be reworded to say that something like "UINodeGraphNode::setSize only available in Ufe preview preview version greater equal to 4.0.100 or 0.5.0"